### ISSUE TITLE
incorrect kernelspec causes failures.

### ISSUE THREAD

**Issue by flamingbear** [nasa/harmony-regression-tests#121]
In reviewing #116, I couldn't run the notebook and and it looked like it was failing to find the correct conda env.

``` shell
./run_notebooks.sh sambah
<[snip]>
RuExecuting:   0%|          | 0/25 [00:00<?, ?cell/s]Kernelspec name papermill-sambah cannot be found!
No such kernel named papermill-sambah
```

Investigation showed ipython notebook had an [unexpected kernelspec](https://github.com/nasa/harmony-regression-tests/pull/116#discussion_r1876878444).

The samba metadata in the ipython notebook was:

``` shell
"kernelspec": {
"display_name": "papermill-sambah",
"language": "python",
"name": "papermill-sambah"
```
vs the spec in a working notebook

``` shell
  "celltoolbar": "Tags",
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
```

Changing the name to python3 enabled the test to run correctly. 

Danny said he had generated he .ipynb file via jupyterlab. I think that's a good place to start, but we should fix this so that it doesn't matter how you generated the iPython notebook test file.

**Comment by flamingbear** [comment]
It seems you can pin the kernel with papermill, https://papermill.readthedocs.io/en/latest/troubleshooting.html 

Not sure if it makes sense to just pin our notebook-entrypoint.sh to python3, but that could resolve this.

**Comment by flamingbear** [comment]
I tested with #299 and it does force the built images to use the python3 kernel, ignoring the metadata from the notebook itself.  I think since we control the build of the containers that run papermill and since this causes no issues when running the notebooks in the browser, we should probably just do this.  The question is do we bump all of the images at once, or just update it in #299 and let each test suite collect the update naturally as they make changes to theirs tests?

I lean towards, adding the change in #299 and letting everyone else pick up the changes naturally as they update their tests. There's no actual work the test owners need to do, the change is baked into the next image when it builds.

TL;DR: When someone updates a notebook from a remote jupyter lab it can get the kernel metadata added to the notebook that prevents our containers from running properly. 
The full change here is to add `-k python3` to the end of the papermill command in notebook-entrypoint.sh to force python3 kernel in the test container.

@owenlittlejohns Any thoughts? happy to slack if that's more efficient.

**Comment by flamingbear** [comment]
I changed my mind again.  I put up a PR just for this, but I definitely want the natural rollout of services picking this up as they make changes to their tests, rather than forcing them all to bump versions for a nothing change.
